.github/workflows/add-labels.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Add labels for area found in bug issue descriptions'
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  issues: write
+
+jobs:
+  add-labels:
+    if: ${{ !github.event.issue.pull_request }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out code
+        uses: actions/checkout@v4
+
+      - name: Add labels for areas found in bug issue descriptions
+        shell: pwsh
+        run: |
+          $match = [regex]::Match('${{ github.event.issue.body }}', '^[#]+ Area\s*?(area:\w+)')
```

Comment:
Not a blocker here, I actually feel this makes it harder to maintain as it cannot easily run on local dev box, editor/IDE don't have good support (e.g. how to debug) （anyways, I'm not the one who will maintain it so feel free to treat it as my subjective opinion)

Reply:
I agree with you on this point! I'm going to circle back to this though. I'm sort of in the middle of improving the release process which is creating a lot of inline powershell scripts. Once that is stable I'll come back and see how it can all be cleaned up.